cmake/README.md

```diff
@@ -98,6 +98,7 @@ There are a number of additional configuration options you can pass to the CMake
 * `-DENABLE_ZSTD=1` : Build the libzstd compressor extension
 * `-DENABLE_SODIUM=1` : Build the libsodium encryptor extension
 * `-DHAVE_DIAGNOSTIC=1` : Enable WiredTiger diagnostics
+* `-DHAVE_REF_TRACK=1` : Enable WiredTiger ref tracking (always enabled in diagnostic build)
```

Comment:
It's not clear what ref tracking is from this text. Maybe "Enable WiredTiger to track recent state transitions for WT_REF structures (always enabled in diagnostic build)"?
The same can be done in base.cmake if we make this change.